frontend/package.json

```diff
@@ -51,5 +53,10 @@
   ],
   "volta": {
     "extends": "../package.json"
+  },
+  "jest": {
```

Comment:
Can we update the `test` script in this file so we can run tests with `yarn test`? Right now it's defined as `"test": "echo \"No test specified\" && exit 0",`. This way we can run all tests in the repo by running `yarn test` from the repo root